stacks/api/bucket/bucket.controller.spec.ts

```diff
@@ -76,7 +77,7 @@ describe("Bucket acceptance", () => {
         CoreTestingModule,
         PassportTestingModule.initialize(),
         DatabaseTestingModule.replicaSet(),
-        BucketModule
+        BucketModule.create(process.env.ENABLE_BUCKET_HOOKS)
```

Comment:
Test targets should not depend on env variables.  I think we'll be better if we put an immediate value instead of a dynamic environment variable.

Reply:
The intention here was to enable hooks based on the modules readiness. Is there any choices that might give us this result?